src/bin/post-comment-to-pr.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import re
+import requests
+
+GITHUB_API = 'https://api.github.com'
+TEST_INSTANCE_PREFIX = 'https://www-dev.greenpeace.org/test-'
+
+
+def get_pull_request(pr_url):
```

Comment:
Can we move the same function in src/bin/book-test-instance.py to a separate file, so we can use it in both places, instead of having to maintain a duplicate?

Reply:
Yes, that's what I meant in the comment with abstracting functions. But I want to do it on second step, since we probably need to adjust `$PYTHONPATH` for imports to work properly. And the function on the other script needs adjustment to be less explicit, since the two functions are returning different things now.

Reply:
My bad I missed that comment :man_facepalming: Yeah I guess that shouldn't block the instance notifcation if this is working already :+1: